redis/reply_test.go

```diff
@@ -225,6 +225,130 @@ func TestSlowLog(t *testing.T) {
 	}
 }
 
+func TestLatency(t *testing.T) {
+	c, err := dial()
+	if err != nil {
+		t.Errorf("TestLatency failed during dial with error " + err.Error())
+		return
+	}
```

Comment:
I would use `testify.require` which makes for a much simpler test flow and is already used in other places:
```suggestion
	c, err := dial()
	require.NoError(t, err)
```

Alternatively `T.Fatalf` achieves what you doing here in one line, with `f` methods it's idiomatic to use formatting parameters. The context will be apparent from the test failing, so no need to name the test for example:
```suggestion
	c, err := dial()
	if err != nil {
		t.Fatalf("dial failed: %s", err)
	}
```